ros/models.py

```diff
@@ -6,12 +6,15 @@
 
 
 class PerformanceProfile(db.Model):
-    __table_args__ = (db.UniqueConstraint('inventory_id', 'report_date'), )
     id = db.Column(db.Integer, primary_key=True)
-    inventory_id = db.Column(UUID(as_uuid=True), nullable=False)
     performance_record = db.Column(JSONB)
     performance_score = db.Column(JSONB)
     report_date = db.Column(db.Date, default=date.today())
+    system_id = db.Column(db.Integer, primary_key=True)
```

Comment:
@apuntamb, do we need to declare `system_id` column as primary_key?
With this change, will you be able to store historical records?

Reply:
To store historical records at a later stage, we can create a new table something like `historical_pp` that can have system_id as the foreign key and store those multiple records with unique dates, does that sound ok?

Reply:
@apuntamb, why do we need `system_id` as primary_key in `PerformanceProfile`?
I think having primary_key on  `(system_id, report_date)` serving the purpose. Sorry but I didn't get the point of creating a separate table `historical_pp` as it will be nothing but the copy of `PerformanceProfile` table.

Reply:
We have a separate PrimaryKeyConstraint specified, so the system_id does not have to have the primary_key annotation